scripts/jenkins/jobs-ibs/cloud-mediacheck.yaml

```diff
@@ -63,6 +64,11 @@
               export RPM_DEPCHECK_MEDIADEPS=" "
               export SRCRPM_DEPCHECK_MEDIADEPS=" "
               export INSTALLCHECK_EXTRA_OPTS="--exclude pattern:"
+          elif [[ "$project" =~ Devel:Cloud:(.*):Appliances ]]; then
+              cloud_version=${BASH_REMATCH[1]}
+              export RPM_DEPCHECK_MEDIADEPS="solv/cloud-$cloud_version-dvd.solv solv/cloud-$cloud_version-updates.solv"
```

Comment:
hmm, why do you add cloud_verison-dvd.solv as a dep? that doesn't make sense to me. the deps iso should not contain anything that depends on cloud 6..